spec/support/data_helpers.rb

```diff
@@ -6,6 +6,29 @@ def create_active_school(*args)
       school_creator.process_new_configuration!
     end
   end
+
+  def create_all_calendar_events
+    term_colour = 'rgb(255,172,33)'
+    inset_day_colour = 'rgb(255,69,0)'
+    inset_day_out_colour = 'rgb(181, 108, 226)'
+    bank_holiday_colour = 'rgb(59,192,240)'
+    holiday_colour = 'rgb(92,184,92)'
```

Comment:
If it's just going to be used in the test then I don't think we need to specify these

Reply:
Agreeeeeeeeeeeeeeed